### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by paulofaria**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+notifications:
+  slack: zewo:VjyVCCQvTOw9yrbzQysZezD1
```
Comment (a2) [inline .travis.yml:2]: ?

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+os:
+  - linux
+  - osx
+language: generic
+sudo: required
+dist: trusty
+osx_image: xcode7.3
+install:
+  - eval "$(curl -sL https://raw.githubusercontent.com/Zewo/Zewo/5254525d9da56df29346fd76e99529c22034d61d/Scripts/install-swiftenv.sh)"
```
Comment (a2) [inline .travis.yml:11]: ?

File: Package.swift [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import PackageDescription
+
+let package = Package(
+    name: "MessagePack",
+    dependencies: [
+        .Package(url: "https://github.com/open-swift/C7.git", majorVersion: 0, minor: 5)
```
Comment (a2) [inline Package.swift:6]: What is C7?

File: README.md [hunk 4]
```diff
@@ -1,31 +1,66 @@
...
-
-
-## Availability
-
-MessagePack.swift is intended for use with Swift 2.0. Compatibility with past and future versions is not guaranteed.
+MessagePack
+==========
+
+[![Travis][travis-image]][travis-url]
+[![Zewo][zewo-image]][zewo-url]
```
Comment (a2) [inline README.md:5]: ?

**Comment by a2** [comment]
I think there are too many commits here to properly address any one in particular. Can you split this PR into smaller PRs?

**Comment by a2** [comment]
Going to close this as there's been no update to the PR.

**Comment by paulofaria** [comment]
@a2 oh man! sorry! created this on accident! haha really sorry!

**Pull request closed without merging**